stencil-library/src/components/business-details/shared-business-details.scss

```diff
@@ -0,0 +1,60 @@
+@import 'bootstrap-utilities';
+@import 'grid';
+
+:host {
+  @include host-base-styles;
+}
+
+// Detail Sections
+detail-section {
+  margin-bottom: 20px;
+
+  &::part(detail-section-title) {
+    font-size: 1.5rem;
+    margin-bottom: 15px;
+  }
+}
+
+// Detail Items
+detail-item {
+  margin-bottom: 10px;
```

Comment:
Can we use the bootstrap grid or utility classes to accomplish this? https://getbootstrap.com/docs/5.3/utilities/spacing/